modules/sandboxed-containers-deploying-workloads-with-kata-runtime-cli.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-cli_{context}"]
+= Deploying a workload in a sandboxed container using the CLI
+
+{sandboxed-containers-first} installs the kata runtime as a secondary optional runtime on the cluster, and not as the primary runtime.
+
+To deploy a pod-templated workload in a sandboxed container, you must add the kata runtime to the workload's YAML file.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+* You have created a `KataConfig` custom resource (CR).
+
+.Procedure
+
+* Add `runtimeClassName: kata` to any pod-templated object:
+** `Pod` objects
+** `ReplicaSet` objects
+** `ReplicationController` objects
+** `StatefulSet` objects
+** `Deployment` objects
+** `DeploymentConfig` objects
+
+.Example for Pod objects
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Pod
+  metadata:
+   name: mypod
+  spec:
+    runtimeClassName: kata
+----
+
+.Example for Deployment objects
+[source,yaml]
+----
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: mypod
+    labels:
+      app: mypod
+  spec:
+    replicas: 3
+    selector:
+      matchLabels:
+        app: mypod
+    template:
+      metadata:
+        labels:
+          app: mypod
+      spec:
+        runtimeClassName: kata
+        containers:
+        - name: mypod
+          image: myImage
+----
+
+The workload is created and {product-title} begins scheduling the workload on nodes enabled with {sandboxed-containers-first}.
```

Comment:
I would remove "The workload is created", or remove the passive voice with:

`{product-title} creates the workloads and begins scheduling it…`

Reply:
Done.